gaia/geo/gdal_functions.py

```diff
@@ -445,6 +445,24 @@ def gdal_zonalstats(zones, raster):
     return list(gen_zonalstats(zones, raster))
 
 
+def gdal_bbox(raster_input):
+    """
+    This function will return bounding box information
+    for a raster layer.
+
+    :param raster_input: raster input filepath
+    :return: list of coordinates in (xmin, ymin, xmax, ymax) format
+    """
+
+    raster = get_dataset(raster_input)
+    geotransform = raster.GetGeoTransform()
+    xmax = geotransform[0] + (raster.RasterXSize * geotransform[1])
+    ymin = geotransform[3] + (raster.RasterYSize * geotransform[5])
+    xmin = geotransform[0]
+    ymax = geotransform[3]
+    return [xmin, ymin, xmax, ymax]
+
```

Comment:
Do you need to handle images that are not aligned with XY axes? In that case, I think you'll need to compute the coordinates for all corners and take the min/max along the X and Y axes.

Reply:
@danlipsa I added the get_footprint function which does what you've asked for.

Reply:
For the parallelogram above (green), xmin for the bounding box (red) is not computed correctly isn't it?

Reply:
@danlipsa You are right. Rasterio's bounds method has the same problem. 
See https://github.com/mapbox/rasterio/issues/525. 

Footprint is computed correctly but bounding box is not for certain cases. I am trying to find a solution for this but not sure if there is a general solution for this problem.